questions-nl.md

```diff
@@ -0,0 +1,167 @@
+# Onze voorstellen en vragen
+
+| <a href="README.md">Home</a> | <a href="introduction-nl.md">Inleiding</a> | Onze voorstellen en vragen | <a href="how-to-respond-nl.md">Reageren op de marktconsultatie</a> |
+| -------- | -------- | -------- | -------- |
+
+Deze pagina bevat drie onderdelen.
+
+1. Een visie voor een open source community - een conceptvisie voor hoe de OpenZaak community eruit zou kunnen zien.
+2. Behoeften van gemeenten - een indicatief overzicht van ondersteuningsvormen die gemeenten nodig hebben van leveranciers.
+3. Vragen marktconsultatie - een overzicht van onze open vragen aan marktpartijen.
+
+## Visie voor een open source community
+
+Wij geloven dat OpenZaak succesvol is als gemeenten en marktpartijen samen werken aan een gemeenschappelijk doel: een open en samenwerkende community die kan schalen.
+
+Voor gemeenten is het belangrijk om een groot en divers leveranciersnetwerk te hebben, die elke gemeente kan ondersteunen met hun specifieke behoeften (inclusief ondersteuning voor ontwikkeling, hosting en implementatie). Dit zal ook bijdragen aan het verminderen van de risico's van gemeenten bij de implementatie van de OpenZaak codebase.
+
+Voor leveranciers is het belangrijk om het aantal deelnemende gemeenten te laten groeien (de markt voor hun diensten te vergroten) en effectief samen te werken met andere leveranciers (het aanvullen van vaardigheden en het verbeteren van de betrouwbaarheid van de implementatie van de OpenZaak codebase).
+
+Zonder alle antwoorden in detail te kennen, omvat onze visie voor de community:
+
+* samenwerken aan één leverancier-neutrale codebase en shared branchingmodel, waarbij uiteenlopende forks worden vermeden
+* collaborative governance, inclusief rond gedeelde roadmaps (feature en technical) en open issue tracker
```

Comment:
beetje lastig: 
governance op basis van samenwerking, waarbij ook wordt samengewerkt rond gedeelde roadmaps etc.

Reply:
Does your comment completely replace the second bullet above? Is the reference to the issue tracker not necessary? (I'll update the English version to match this one.)

Reply:
Nee, het stukje "(feature en technical) en open issue tracker" moet er wel bij

Reply:
open issue tracker --> openbare issue tracker